### DISCUSSION TITLE
如何设计排行榜规则？

### DISCUSSION THREAD

**Discussion by ccxdcyl** [decert-me/website#290] [General]
你是想成为第一个拿到NFT的人，还是得到最高分的人呢？那么规则怎么定呢？

下面是挑战排行榜规则：

- 显示参与人数：参与挑战的人数
- 两个榜单：神速榜、高分榜
- 神速榜：根据完成挑战（领取NFT）的时间，最先领取的在最前
   榜单内容：用户头像、地址（缩写）、完成的时间 
- 高分榜：根据完成挑战得分（如果挑战多次，按最高得分算 ），相同得分，根据挑战时间排序。
   榜单内容：用户头像、地址（缩写）、得分、得分时间

有不足之处或者其他想法都可以在评论区回复～～～

**Comment by xilibi2003** [comment]
我觉得有 3 个维度：
1. 最先完成挑战
2. 最快完成挑战
3. 最好挑战成绩
